.github/workflows/continuous-integration.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         path: node_modules
         key: ${{ runner.os }}-node_modules-${{ hashFiles('yarn.lock') }}
     - name: Install
-      run: yarn install --immutable
+      run: yarn install --immutable && yarn playwright install
```

Comment:
We don't actually need Firefox or WebKit at present. We could use `yarn playwright install chromium` to install only the browser we use. This will save some time and disk space. If someone wants to install additional browsers for testing they can do so.

Reply:
Oh, that can be done? I was googling to see if that was possible and didn't find anything. I'll change it.